src/batches/setBatchCoregistrationFuncToAnat.m

```diff
@@ -1,7 +1,24 @@
 % (C) Copyright 2019 CPP BIDS SPM-pipeline developers
 
 function matlabbatch = setBatchCoregistrationFuncToAnat(matlabbatch, BIDS, subID, opt)
-
+  %
+  % Short description of what the function does goes here.
+  %
+  % USAGE::
+  %
+  %   [argout1, argout2] = templateFunction(argin1, [argin2 == default,] [argin3])
+  %
+  % :param argin1: (dimension) obligatory argument. Lorem ipsum dolor sit amet,
+  %                consectetur adipiscing elit. Ut congue nec est ac lacinia.
```

Comment:
haha some latin to practise @Remi-Gau ?

Reply:
Actually `Lorem ipsum` is a very useful trick to use filler text when you just want to work on the layout of a page and not worry about the content.

https://www.lipsum.com/

I am sure there must be a python package to generate entire pages of it.